anthos-vmruntime/README.md

```diff
@@ -159,33 +159,42 @@ create the VM using `kubectl apply -f pos-vm.yaml`.
 kubectl virt create vm pos-vm \
 --boot-disk-size=80Gi \
 --memory=4Gi \
---cpu=2 \
+--vcpus=2 \
 --image=https://storage.googleapis.com/pos-vm-images/pos-vm.qcow2
 ```
 
 ```sh
 # expected output
-Constructing yaml for vm "pos-vm":
-Deployment yaml for vm "pos-vm" is saved to pos-vm.yaml.
-Apply yaml for vm "pos-vm"
-Creating boot DataVolume "pos-vm-boot-dv"
-Creating gvm "pos-vm"
+Constructing manifest for vm "pos-vm":
+Manifest for vm "pos-vm" is saved to /home/tfadmin/google-virtctl/pos-vm.yaml
+Applying manifest for vm "pos-vm"
+Created gvm "pos-vm"
 ```
 > **Note:** After you run `kubectl virt create vm pos-vm`, the CLI would have
 > created a yaml file named after the vm (`pos-vm.yaml`). You can inspect it to
-> see the definiton of the `VirtualMachine` and `DataVolume`.
+> see the definiton of the `VirtualMachine` and `VirtualMachineDisk`.
 
 ---
 ### Check the VM creation status
 
-Creation of the VM requires two resources to be created: `DataVolume` and
-`VirtualMachine`. `DataVolume` is the _persistent disk_ where the contents of
-the **VM image** is imported into and is made bootable from. `VirtualMachine` is
-the **VM template** created based off of the image. The `DataVolume` is mounted
-into the `VirtualMachine` before the VM is booted up. Creation of a
-`VirtualMachine`, automatically triggers the creation of a
-`VirtualMachineInstance` resource which represents a _RUNNING_ instance of a VM.
+Creation of the VM requires two resources to be created: `VirtualMachineDisk` and
+`VirtualMachine`. `VirtualMachineDisk` is the _persistent disk_ where the
+contents of the **VM image** is imported into and is made bootable from.
+`VirtualMachine` is the **VM template** created based off of the image. The
+`VirtualMachineDisk` is mounted into the `VirtualMachine` before the VM is
+booted up. Creation of a `VirtualMachine`, automatically triggers the creation
+of a `VirtualMachineInstance` resource which represents a _RUNNING_ instance of
+a VM.
```

Comment:
I think we should add a few more details so readers are not confused of the use of `DataVolume` below. Also I have formatted it a bit.

```suggestion
Creation of the VM requires two resources to be created: `VirtualMachineDisk` and
`VirtualMachine`. 

- `VirtualMachineDisk` is the _persistent disk_ where the contents of the
**VM image** is imported into and is made bootable from. The `VirtualMachineDisk`
wraps around the `DataVolume` resource which can be used to monitor progress
of the import.

- `VirtualMachine` is the **VM template** created based off of the image. 
Creation of a `VirtualMachine`, automatically triggers the creation of a
`VirtualMachineInstance` resource which represents a _RUNNING_ instance of
a VM. The `VirtualMachineDisk` is mounted into the `VirtualMachine` before the VM is
booted up. 
```

Reply:
LGTM